pkg/api/controller.go

```diff
@@ -3424,12 +3424,20 @@ func (c *Controller) RevertBranch(w http.ResponseWriter, r *http.Request, body a
 		writeError(w, r, http.StatusUnauthorized, "user not found")
 		return
 	}
-	err = c.Catalog.Revert(ctx, repository, branch, catalog.RevertParams{
+	revertParams := catalog.RevertParams{
 		Reference:    body.Ref,
 		Committer:    user.Committer(),
 		ParentNumber: body.ParentNumber,
 		AllowEmpty:   swag.BoolValue(body.AllowEmpty),
-	}, graveler.WithForce(swag.BoolValue(body.Force)))
+	}
+	if body.CommitOverrides != nil {
+		revertParams.Overrides = graveler.CommitOverrides{
+			Message:  *body.CommitOverrides.Message,
```

Comment:
That what I was worried about:
1. First of all, this could be nil pointer exception.
2. If I just want to change the message, now I need to pass the metadata and the date?

Reply:
I think the answer to both is "no", will verify tomorrow.
My intention is for the answer to be "no" anyway, so if it's not the answer I'll make it the answer :slightly_smiling_face:

Reply:
I added tests for partial overrides, and `nil` overrides that make sure there's no NPE, and "revert/cherry-pick successful" pass in empty overrides struct (as opposed to `nil`) - so we've got good coverage there :slightly_smiling_face: